CHANGELOG.md

```diff
@@ -202,6 +202,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed `_compare_version` for python packages ([#10762](https://github.com/PyTorchLightning/pytorch-lightning/pull/10762))
 
 
+- Fixed passing `Accelerator` objects to the accelerator flag with `devices=x` ([#10773](https://github.com/PyTorchLightning/pytorch-lightning/pull/10773))
+
+
+- Resolve training type plugin when passed within `Accelerator` ([#10773](https://github.com/PyTorchLightning/pytorch-lightning/pull/10773))
```

Comment:
I am confused by this fix. IMO it is not worth doing that considering the refactors that happen right now for 1.6. I could however see this being included in 1.5.x

Reply:
@awaelchli But it would be required for 1.5.x, right?

Reply:
Yes sure, it can be merged to master first, then be included in 1.5.x, but then tests that you add here become mostly outdated immediately after. That's all I'm saying.

Reply:
In that case, let's just change the target branch to the bugfix release branch